purchase_order_type/__manifest__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
@oihane Copyright should be 

```
# Copyright (C) 2015 Camptocamp SA
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Reply:
I didn't have any idea if I could change it.